mifosng-android/src/main/java/com/mifos/utils/CheckSelfPermissionAndRequest.java

```diff
@@ -128,8 +131,16 @@ public void onClick(DialogInterface dialog, int which) {
                                                 R.string.package_name), activity.getPackageName()
                                                 , null);
                                         intent.setData(uri);
-                                        activity.startActivityForResult(intent,
-                                                Constants.REQUEST_PERMISSION_SETTING);
+                                        try {
```

Comment:
It is crashing on my device (Lenovo k4). I'll change the approach to use the package manager. :+1: 
Thanks.